packages/gridfs/package.js

```diff
@@ -15,12 +15,12 @@ Package.onUse(function(api) {
   api.versionsFrom('1.0');
 
   api.use(['cfs:base-package@0.0.27', 'cfs:storage-adapter@0.1.1']);
-  api.addFiles('gridfs.server.js', 'server');
-  api.addFiles('gridfs.client.js', 'client');
+  api.addFiles('packages/gridfs/gridfs.server.js', 'server');
+  api.addFiles('packages/gridfs/gridfs.client.js', 'client');
 });
 
 Package.onTest(function(api) {
   api.use(['cfs:gridfs', 'test-helpers', 'tinytest'], 'server');
-  api.addFiles('tests/server-tests.js', 'server');
-  api.addFiles('tests/client-tests.js', 'client');
+  api.addFiles('packages/gridfs/tests/server-tests.js', 'server');
+  api.addFiles('packages/gridfs/tests/client-tests.js', 'client');
```

Comment:
when I try to `$ meteor publish-for-arch cfs:gridfs@0.0.28` I get: "Malformed source tarball"
cc. @ekatek